redirector.py

```diff
@@ -36,7 +36,7 @@
 from tornado import gen, web
 from tornado import ioloop
 
-from tornado.httpclient import HTTPRequest, HTTPError, AsyncHTTPClient
```

Comment:
You can delete the `HTTPClient` from the import now that you're using the `AsyncHTTPClient`